src/clr-angular/popover/dropdown/_dropdown.clarity.scss

```diff
@@ -130,7 +130,7 @@
             }
 
             &.expandable {
-                $clr-dropdown-caret-xOffset: 0.333rem;
+                $clr-dropdown-caret-margin-top: 0.375rem;
```

Comment:
Can we move this to `./_variables.dropdown.scss` ?

Reply:
I don't think we can because its a local variable. It changes in different breakpoints. I would have to create multiple variables if we want to do that. For now, I think this is fine. The whole dropdown css needs a refactor and we can get this fixed that time. Does that sound okay?